utilities/src/main/java/io/syndesis/qe/bdd/validation/SfValidationSteps.java

```diff
@@ -216,7 +218,7 @@ public void verifyLeadCreatedResponse(String email, String queueName) {
     @Then("^verify that lead was deleted$")
     public void verifyLeadRemoval() {
         // Add a delay for the integration processing
-        TestUtils.sleepIgnoreInterrupt(5000L);
+        TestUtils.sleepIgnoreInterrupt(30000L);
```

Comment:
Why the longer wait time?

Reply:
no particular reason, just to be sure it won't fail because it took the integration a bit of time... probably could be replaced with a smarter wait :grin:

Reply:
@tplevko should be better now